packages/sui-studio-utils/src/domain-builder/index.js

```diff
@@ -1,15 +1,18 @@
+/* eslint-disable no-nested-ternary */
 export default class DomainBuilder {
   static extend({domain = {}}) {
     return new DomainBuilder({domain})
   }
 
+  // @ts-ignore
```

Comment:
¿?

Reply:
The linter is complaining about this...
deleted